internal/e2e/rollup/bundle-umd_golden.js_

```diff
@@ -9,22 +9,24 @@
   (factory((global.bundle = {}),global.runtime_name_of_global_var));
 }(this, (function (exports,some_global_var) { 'use strict';
 
-  const fum = 'Wonderland';
+  var fum = 'Wonderland';
 
   var hello = 'Hello';
 
-  const name = 'Alice';
-
-  console.log(`${hello}, ${name} in ${fum}`);
+  var name = 'Alice';
 
+  console.log(hello + ", " + name + " in " + fum);
   // Test for sequences = false
-  class A {
-    a() {
-      return document.a;
-    }
-  }
+  var A = /** @class */ (function () {
```

Comment:
I need to study this a bit to understand why the syntax is changing here. It seems like the previous output was just wrong, and should never have had a class keyword?

Reply:
Yes that is a good point, I'll see if I can have a look into that myself the next few days.